src/synchronisation.sh

```diff
@@ -338,6 +340,7 @@ Delay between subsequent checks for existence of file. Default 1.
 
 =back
 =cut
+#'
```

Comment:
what are these #' for?

Reply:
That's part of the commit "comment fixes for dummy editors". Some browsers have difficulties with text highlighting were the apostrophes are in the redirected POD docs.